guides/plugins/apps/administration/add-custom-action-button.md

```diff
@@ -74,6 +75,18 @@ Again you can verify the authenticity of the incoming request, like with [webhoo
 This feature was added in Shopware 6.4.3.0, previous versions will ignore the response content.
 {% endhint %}
 
+{% hint style="info" %}
+Starting from Shopware version 6.4.8.0, the requests of the [tab](#opening-a-new-tab-for-the-user) and [custom modal](#open-a-custom-modal) will have the following additional query parameters:
```

Comment:
```suggestion
Starting from Shopware version 6.4.8.0, the requests of the [tab](#opening-a-new-tab-for-the-user) and [custom modal](#open-a-custom-modal) have the following additional query parameters:
```

Reply:
Shouldn't it be "[...] and custom modal **HAS** the following additional [...]"?